packages/expo-apple-authentication/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "expo-apple-authentication",
+  "version": "1.0.0",
+  "description": "A unimodule that provides SignIn With Apple",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "sideEffects": false,
+  "scripts": {
+    "build": "expo-module build",
+    "clean": "expo-module clean",
```

```suggestion
    "clean": "expo-module clean",
    "lint": "expo-module lint",
```
And also run "yarn prepare" to regen the eslint config file.